app/Actions/ConfigureFrontendFramework.php

```diff
@@ -2,36 +2,77 @@
 
 namespace App\Actions;
 
-use App\Shell\Shell;
+use App\Shell;
+use Illuminate\Support\Facades\Config;
 
 class ConfigureFrontendFramework
 {
     use LamboAction;
 
     private $shell;
-
     private $laravelUi;
+    private $availableFrontends;
 
     public function __construct(Shell $shell, LaravelUi $laravelUi)
     {
         $this->shell = $shell;
         $this->laravelUi = $laravelUi;
     }
 
-    public function __invoke()
+    public function __invoke(array $availableFrontends)
     {
-        if (! config('lambo.store.frontend')) {
+        $this->availableFrontends = $availableFrontends;
+
+        if (! Config::get('lambo.store.frontend')) {
```

Comment:
Remember to remove all use of the config facade and replace, for now, with the global helper